views/aggregate-flow-metadata.json

```
@@ -1612,12 +1644,18 @@
                     }
                 }
             },
-            "install_command": "pip install prefect-databricks",
-            "description": {
-                "summary": "Triggers a Databricks jobs runs and waits for the triggered runs to complete."
-            }
+            "path_containing_flow": "prefect_databricks/flows.py",
+            "slug": "jobs_runs_submit_and_wait_for_completion"
         },
         "Wait for completion of jobs runs": {
+            "collection_repo_url": "https://github.com/PrefectHQ/prefect-databricks",
+            "description": {
+                "summary": "Waits for the jobs runs to finish running"
+            },
+            "documentation_url": "https://prefecthq.github.io/prefect-databricks/flows/#prefect_databricks.flows.Wait for completion of jobs runs",
```

This needs to be the slug instead.
```suggestion
            "documentation_url": "https://prefecthq.github.io/prefect-databricks/flows/#prefect_databricks.flows.Wait for completion of jobs runs",
```

---

eek true